packages/ruleset/src/functions/parameter-case-convention.js

```diff
@@ -0,0 +1,84 @@
+const { casing } = require('@stoplight/spectral-functions');
+const { isSdkExcluded, isDeprecated } = require('../utils');
+
+// Error message prefix for each parameter type.
+const errorMsgPrefix = {
+  query: 'Query parameter names ',
+  path: 'Path parameter names ',
+  header: 'Header parameter names '
+};
```

Comment:
The [OpenAPI spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#parameter-object) also supports a parameter type of "cookie" - I'm guessing we don't have any prescriptions about this type but I wanted to make sure 🍪

Reply:
Correct, we were actually only checking case conventions for query and path params in the old rule, so I added checking for header params in the new rule.
In the SDK generator, we don't do anything with cookie-type params so I figured I'd steer clear of those in the validator.
I actually haven't seen an API yet for an IBM Cloud service that uses cookie params anyway :)